newsrevenuehub/rev-engine
# Dev 488 change donation to contribution

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

To rename user-visible mentions of "Donations" to "Contributions"

#### What's this PR do?

- Replaces "Donations" on the Org Portal tab with "Contributions"
- Replaces mentions of Donations with Contributions in the README

#### Why are we doing this? How does it help us?

- Contributions is broader than donations. This will make our personas and other terms more consistent.

#### How should this be manually tested?

- Look at the Org Admin tabs and confirm you see "Contributions" and not "Donations"

#### Have automated unit tests been added?

No. Just copy changes.

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

I saw at least one possible URL as `/donations/` that I didn't change because I wasn't sure of the implications or what the potentially broken link would mean. Opened as a separate ticket: [DEV-488]

#### Has this been documented? If so, where?

No need, I don't think.

#### What are the relevant tickets?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**

[DEV-488]

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No.

#### Are there next steps? If so, what? Have tickets been opened for them?
**(Note: Please use syntax [JIRA-123] to link any relevant tickets)**

Yes: 
- https://news-revenue-hub.atlassian.net/browse/DEV-1446
- https://news-revenue-hub.atlassian.net/browse/DEV-1464 

[DEV-488]: https://news-revenue-hub.atlassian.net/browse/DEV-488?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
